test/AutoDiff/validation-test/simple_math.swift

```diff
@@ -1,7 +1,11 @@
 // RUN: %target-run-simple-swift
+
```

Comment:
We can add more tests regarding "multiple result indices" (e.g. differentiability witness SILGen tests) when landing TF-1288: relaxing `@differentiable` and `@derivative` type-checking to allow original declarations with multiple semantic results.